packages/chats/src/Lightbox/DuplicateContent.tsx

```diff
@@ -169,7 +169,10 @@ export const DuplicateContent = ({
                             <View onLayout={onLayout}>{previewImage}</View>
                         </Animated.View>
                         {isFullSizeDisplayed ? (
-                            <Animated.View style={styles.fullSizeImage}>
+                            <Animated.View
+                                testID={`image_expanded`}
```

Comment:
You ESLint seem not to work properly in `surf@6.4.4` ;)
```suggestion
                                testID="image_expanded"
```